web/portal/brand/src/router/AppRouteContentWrapper/AppRouteContentWrapper.tsx

```diff
@@ -1,18 +1,19 @@
 import { Box, Toolbar } from '@mui/material';
 import { Header, Menu, Footer } from '@irontec/ivoz-ui/components/layout';
 import { RouteMap } from '@irontec/ivoz-ui/router/routeMapParser';
-import { languages } from '../../translations/languages';
+import { Language } from '@irontec/ivoz-ui/store/i18n';
```

Comment:
This is not necessary anymore as Header will read languages from the store itself